libs/gltfio/src/AssetLoader.cpp

```diff
@@ -605,6 +605,12 @@ void FAssetLoader::createRenderable(const cgltf_data* srcAsset, const cgltf_node
         }
         mRenderableManager.setMorphWeights(renderable, weights.data(), size);
     }
+    else
+    {
+        const float initWeights[1]={0};
+        RenderableManager::Instance renderable = mRenderableManager.getInstance(entity);
+        mRenderableManager.setMorphWeights(renderable, initWeights, 1);
+    }
```

Comment:
Is that part of the change absolutely needed too? I really don't like that it's in the client code -- that means that we need to *document* this behavior.

Is there a way to do this inside the filament library instead?

Reply:
Hi, @pixelflinger , thanks for the review and the recommendation.

1. The morphWeights uniform array is also needed to be initialized to get rid of this crash on some Adreno gpu phones. Any models with mesh downloaded from mixamo website would trigger this crash, since there's no morph target, but there's bone skinning. For example, use any model with mesh from this link(in my case, my phone is with Adreno (TM) 506):
https://www.mixamo.com/#/?page=1&query=danc

I moved the code change to a suitable place in RenderableManager.cpp, I tested it and it still works.

2. I added a workaround just as you said. Since the renderer is already recorded in mContext.state, I added a get method to get it. I also added some short comments in the code.